plugins/misp/plugin.spec.yaml

```diff
@@ -589,21 +623,25 @@ actions:
       event:
         title: Event ID
         description: Event ID to append to
+        example: "1"
         type: string
         required: false
       urls:
         title: URLs
         description: URLs to add
+        example: ["https://example.com"]
         type: '[]string'
         required: false
       comment:
         title: Comment
         description: Attribute comment
+        example: "Example comment"
         type: string
         required: false
       distribution:
         title: Distribution
         description: Distribution type
+        example: "All Communities"
```

Comment:
Since it is enum, please update example to a valid enum

Reply:
Line 650, there is an enum like `All Communities`